runtime/themes/ayu_light.toml

```
@@ -38,16 +38,17 @@
 "ui.cursor.match" = { fg = "orange" }
 "ui.linenr" = { fg = "dark_gray" }
 "ui.linenr.selected" = { fg = "orange" }
-"ui.statusline" = { bg = "black" }
+"ui.cursorline" = { bg = "black" }
+"ui.statusline" = { fg = "foreground", bg = "black" }
 "ui.popup" = { bg = "black" }
 "ui.window" = { fg = "dark_gray" }
-"ui.help" = { bg = "black" }
+"ui.help" = { fg="foreground", bg = "black" }
```

```suggestion
"ui.help" = { fg  = "foreground", bg = "black" }
```
If possible, please stick to the original format of the file, but I think you probably missed some.

---

Thanks! I will update my linter to require fg and bg for help, and rerun.